packages/foreman/foreman/foreman.spec

```diff
@@ -57,6 +57,7 @@ Requires: %{name}-debug
 
 # start specfile main Requires
 Requires: %{?scl_prefix}rubygem(rails) = 5.2.1
+Requires: %{?scl_prefix_ruby}rubygem(bigdecimal)
```

Comment:
This section is autogenerated using [update-requirements](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements) so the next update will wipe this out. If it's needed, either add it in a section that's not autogenerated or add it to https://github.com/theforeman/foreman/blob/develop/Gemfile